client/my-sites/patterns/pages/category/index.tsx

```diff
@@ -106,13 +110,13 @@ export const PatternsCategoryPage = ( {
 					<ToggleGroupControl label="" isBlock className="patterns__toggle-view" value="patterns">
 						<ToggleGroupControlOption
 							value="patterns"
-							label="List view"
-							className="patterns-button__list-view"
+							label={ ( <Icon icon={ iconCategory } size={ 20 } /> ) as unknown as string }
```

Comment:
Wow, that's annoying 😅 I see why you did it, though. Let's keep it as is.

Reply:
Yeah, I was "dancing around" 😅 and it's only one good way :(